telegram/messageentity.py

```diff
@@ -55,7 +55,7 @@ def __init__(self, type, offset, length, url=None, user=None, **kwargs):
         self.url = url
         self.user = user
 
-        self._id_attrs = (self.type, self.offset, self.length)
+        self._id_attrs = (self.type, self.offset, self.length, self.url, self.user)
```

Comment:
Like I mentioned in chat, I prefer to remove the `self.url` and `self.user`.
type, offset and length are the unique identifiers for a MessageEntity.

Reply:
@Eldinnie Reverted